xtask/src/qemu.rs

```diff
@@ -394,14 +388,14 @@ pub fn run_qemu(arch: UefiArch, opt: &QemuOpt) -> Result<()> {
             // Map the QEMU exit signal to port f4.
             cmd.args(["-device", "isa-debug-exit,iobase=0xf4,iosize=0x04"]);
 
-            // OVMF debug builds can output information to a serial `debugcon`.
-            // Only enable when debugging UEFI boot.
-            // cmd.args([
-            //     "-debugcon",
-            //     "file:debug.log",
-            //     "-global",
-            //     "isa-debugcon.iobase=0x402",
-            // ]);
+            // We also add a debugcon logger next to the serial one, as
+            // it sumplifies debugging in some rare cases.
```

Comment:
```suggestion
            // it simplifies debugging in some rare cases.
```

Reply:
Weird. I am 100% sure I already changed that; Might have been on my other laptop and I forgot to push? Thanks